docs/stories/Box.stories.tsx

```diff
@@ -12,44 +12,83 @@ type Story = StoryObj<typeof Box>;
 
 export const Base = {
   render: () => (
-    <Box padding={4} background="primary700" shadow="filterShadow" borderColor="danger600">
+    <Box padding={4} margin={2} background="primary700" shadow="filterShadow" borderColor="danger600">
       <Typography textColor="neutral0">Hello world</Typography>
     </Box>
   ),
 
   name: 'base',
 } satisfies Story;
 
-export const ResponsiveSpaces = {
+export const ResponsiveStyles = {
   render: () => (
     <Box
-      padding={[11, 6, 1]}
-      background="primary700"
-      shadow="filterShadow"
-      hiddenXS
-      borderColor="danger600"
+      paddingTop={{ initial: '4rem', medium: '14px', large: 6 }}
+      padding={['2rem', '4rem', '8rem']}
+      marginTop={{ initial: 1, medium: '1rem', large: '22px' }}
+      margin={{ initial: 2, medium: [2, '4px'], large: [6, '4px', 8] }}
```

Comment:
I'm a bit confused by the array breakpoint types and how they're applied.

I'm understanding it as e.g. the large breakpoint here would become:

'margin-block-start': 6, 'margin-inline-end': '4px', 'margin-block-end': 8, 'margin-inline-start': 6 (again?)

is that right?